babel.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function(api) {
+api.cache(true);
+
+  const presets = ["@babel/preset-env"];
+  const plugins = ["@babel/plugin-transform-react-jsx"];
```

Comment:
Why not just write
```js
module.exports = {
  presets: ["@babel/preset-env"],
  plugins: ["@babel/plugin-transform-react-jsx"]
}
```
?

Reply:
This looks new for Babel 7. Apparently setting the cache to true keeps the config in memory for each calls instead of reading it each time. See https://babeljs.io/docs/en/config-files#apicache. Both work. I don't care which one is preferred ;) I simply did it the way it is documented.

Reply:
My bad, I mean

```js
return {
  presets: ["@babel/preset-env"],
  plugins: ["@babel/plugin-transform-react-jsx"]
}
```
without unnecessary variables